### DISCUSSION TITLE
Oh The Biomes We've Gone

### DISCUSSION THREAD

**Discussion by supermassimo** [DynamicTreesTeam/DynamicTrees#934] [Compatibility Addon Suggestions]
https://www.curseforge.com/minecraft/mc-mods/oh-the-biomes-weve-gone
+ Tens of biomes
+ Tens of trees
Update from: https://www.curseforge.com/minecraft/mc-mods/dynamic-trees-byg

**Comment by J-Ray17** [comment]
This would be absolutely awesome! I really hope this actually comes to be

**Comment by supermassimo** [comment]
Done :) https://www.curseforge.com/minecraft/mc-mods/dynamic-trees-bwg

**Reply by DeshiDesu** [reply]
W congrats on publishing

**Comment by kirakenjiro** [comment]


**Reply by supermassimo** [reply]
What other versions are there to port this addon to?

**Reply by supermassimo** [reply]
Dynamic trees is only available on forge 1.20.1 and neo forge 1.21.1 so only those versions are options to port add-ons to.
Once the fabric 1.21.1 version is made then it will be an option too
